Core/src/org/sleuthkit/autopsy/datamodel/FileTypesByExtension.java

```diff
@@ -418,17 +421,18 @@ public void update(Observable o, Object arg) {
         }
 
         @Override
-        protected boolean createKeys(List<Content> list) {
+        protected boolean createKeys(List<FileTypesKey> list) {
             try {
-                list.addAll(skCase.findAllFilesWhere(createQuery(filter)));
+                list.addAll(skCase.findAllFilesWhere(createQuery(filter))
+                        .stream().map(f -> new FileTypesKey(f)).collect(Collectors.toList()));
```

Comment:
to be even more concise you could use a method reference to the constructor:

...map(FileTypesKey::new).collect....